modules/virt-subscribing-cli.adoc

```diff
@@ -19,26 +19,26 @@ To subscribe, configure `Namespace`, `OperatorGroup`, and `Subscription` objects
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: openshift-cnv
+  name: {CNVNamespace}
 ---
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: kubevirt-hyperconverged-group
-  namespace: openshift-cnv
+  namespace: {CNVNamespace}
 spec:
   targetNamespaces:
-    - openshift-cnv
+    - {CNVNamespace}
```

Comment:
Sorry, I missed that one.
In case of the community version of CNV, the operator is defined with AllNamespaces InstallMode, which means its OperatorGroup targets all namespaces, in that case, the spec section can be left empty:
```
spec: {}
```
see:
https://docs.openshift.com/container-platform/4.13/operators/understanding/olm/olm-understanding-operatorgroups.html
> Alternatively, you can omit both spec.selector and spec.targetNamespaces to specify a global Operator group, which selects all namespaces:

Reply:
hi @aspauldi ,
yes, it looks good. thanks!